RestrictEvents/SoftwareUpdate.cpp

```diff
@@ -195,20 +195,26 @@ static sysctl_oid *sysctl_by_name(sysctl_oid_list *sysctl_children, const char *
     return nullptr;
 }
 
+static const char revasset_arg = checkKernelArgument("-revasset");
```

Comment:
Please do not execute this in constructors. It is way too evil as the code runs before anything else, and we cannot easily control Lilu APIs before that. Do it in PluginConfiguration startup code.